summarization/text_media_matching/text_media_matcher.py

```diff
@@ -7,9 +8,16 @@
 
 class TextMediaMatcher:
     '''Class to integrate the TextMediaMatching utilities'''
-    def __init__(self, text_contents, media_contents):
+
+    def __init__(self, text_contents, media_contents,
+                 distance_metric_type="absolute-difference"):
         self.text_contents = text_contents
         self.media_contents = media_contents
+        self.distance_metric_type = distance_metric_type
+        if distance_metric_type not in [
+            "absolute-difference",
+                "signed-difference"]:
+            raise IncorrectInputError('distance_metric_type argument is wrong')
```

Comment:
Since this is called internally, maybe we don't need this check.

I guess this is an example of why to prefer enums over strings (not sure if this is valid in python because of no type safety) I think we do this in stamp_page_picker also - we'll do a code cleanup after this and revisit this then.

Reply:
Thanks for the suggestion. Removing the check accordingly.